### PR TITLE
fix: update article for JavaScript String.indexOf()

### PR DESCRIPTION
**Pull request by HasanAbdullah31**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/javascript/standard-objects/string/string-prototype-indexof/index.md [hunk 1]
```diff
@@ -2,35 +2,42 @@
...
 'Blue Whale'.indexOf('Whale', 7); // returns -1
 'Blue Whale'.indexOf('');         // returns  0
 'Blue Whale'.indexOf('', 9);      // returns  9
 'Blue Whale'.indexOf('', 10);     // returns 10
 'Blue Whale'.indexOf('', 11);     // returns 10
-'Blue Whale'.indexOf('blue');     // returns -1
 ```
 
 ### More Information:
 - MDN documentation: <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf' target='_blank' rel='nofollow'>MDN</a>
```
Comment (moT01) [inline guide/english/javascript/standard-objects/string/string-prototype-indexof/index.md]: [String.prototype.indexOf() on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf)

**Review by moT01 — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@moT01 Is this PR merge-ready?

**Review by thecodingaviator — APPROVED** [review]
LGTM. @moT01 merge at discretion

**Review by moT01 — APPROVED** [review]
Looks good @HasanAbdullah31 - thanks for contributing to freeCodeCamp! 🎉

**Pull request merged**